sycl/doc/extensions/experimental/sycl_ext_oneapi_cuda_tex_cache_read.asciidoc

```diff
@@ -50,10 +50,14 @@ change incompatibly in future versions of {dpcpp} without prior notice.
 *Shipping software products should not rely on APIs defined in this
 specification.*
 
+== Backend support status
+
+This extension is supported by all backends. However, as described in the following sections, this extension is currently only functionally useful in the `ext_oneapi_cuda` backend.
+
 
 == Overview
 
-This extension exposes the https://docs.nvidia.com/cuda/parallel-thread-execution/index.html#data-movement-and-conversion-instructions-ld-global-nc[ld-global-nc] ptx instruction so that users can load a register variable to the non-coherent read only texture cache. The texture cache is designed for random access reads and is most performant when work-items within a sub-group read a set of addresses of texture memory that are close to one another in the cache.
+When used with the `ext_oneapi_cuda` backend only, this extension exposes the https://docs.nvidia.com/cuda/parallel-thread-execution/index.html#data-movement-and-conversion-instructions-ld-global-nc[ld-global-nc] ptx instruction so that users can load a register variable to the non-coherent read-only texture cache. The texture cache is designed for random access reads and is most performant when work-items within a sub-group read a set of addresses of texture memory that are close to one another in the cache. This extension introduces a free function, `__ldg`, that is available for all backends and devices; however this function will only make use of a special hardware feature (The texture cache) when called from the `ext_oneapi_cuda` backend. The only reason that `__ldg` is supported on backends other than `ext_oneapi_cuda` is to allow for code portability.
```

Comment:
```suggestion
When used with the `ext_oneapi_cuda` backend only, this extension exposes the https://docs.nvidia.com/cuda/parallel-thread-execution/index.html#data-movement-and-conversion-instructions-ld-global-nc[ld-global-nc] ptx instruction so that users can load a register variable to the non-coherent read-only texture cache. The texture cache is designed for random access reads and is most performant when work-items within a sub-group read a set of addresses of texture memory that are close to one another in the cache. This extension introduces a free function, `+__ldg+`, that is available for all backends and devices; however this function will only make use of a special hardware feature (The texture cache) when called from the `ext_oneapi_cuda` backend. The only reason that `+__ldg+` is supported on backends other than `ext_oneapi_cuda` is to allow for code portability.
```

The formatting is messed up in the HTML render.  I'm guessing that adding `+` ([inline passthrough](https://docs.asciidoctor.org/asciidoc/latest/pass/pass-macro/)) as I show here will fix it.  If this doesn't work, try adding the `+` outside of the backticks.

Reply:
Oh yes, dollar signs work: I added some more `__ldg`s and forgot about this!.